src/App.css

```diff
@@ -1,3 +1,7 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
```

Comment:
🔧 The font is setup in the index.html file. There shouldn't be any need to do it here too.
https://github.com/aaron-org/snapit-hacktoberfest/blob/main/index.html

Reply:
Addressed:  7526db14569b6afaed4daf005a316c3ff84f140d